From: uazo <uazo@users.noreply.github.com>
Date: Sat, 28 Jun 2025 08:17:35 +0000
Subject: Keep disabled cache sharing for pervasive scripts

Keeps disabled the new feature that allows bypassing the cache partition for specific urls.

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 .../features_cc/Disable-Cache-Sharing-For-PervasiveScripts.inc   | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 cromite_flags/services/network/public/cpp/features_cc/Disable-Cache-Sharing-For-PervasiveScripts.inc

diff --git a/cromite_flags/services/network/public/cpp/features_cc/Disable-Cache-Sharing-For-PervasiveScripts.inc b/cromite_flags/services/network/public/cpp/features_cc/Disable-Cache-Sharing-For-PervasiveScripts.inc
new file mode 100644
--- /dev/null
+++ b/cromite_flags/services/network/public/cpp/features_cc/Disable-Cache-Sharing-For-PervasiveScripts.inc
@@ -0,0 +1 @@
+SET_CROMITE_FEATURE_DISABLED(kCacheSharingForPervasiveScripts);
--
